Fix/ro 2966 vis plott i bildekarusell

gruble opened the pull request:
Se saken (https://nveprojects.atlassian.net/browse/RO-2966)
Denne krever [denne endringen i API'et](https://dev.azure.com/NVE-devops/Varsom%20Regobs/_git/regobs/pullrequest/12873), så ikke fullfør før vi har rullet ut den endringen i alle miljøer.
Du kan teste det ved å kjøre opp API-PR'en lokalt. 

Henter nå url til bildet av plottet fra `PlotImage` i snøprofil-viewModel. API'et legger nå på beskrivelsen til bildet hvis det er noen, så da slipper vi det her.
I tillegg har vi fått et ekstra felt, `isSnowProfile` i Attachment, så vi slipper å trikse med ekstra felter for dette i klienten for å skille mellom snøprofil-plottet og de andre bildene.

gruble committed:
startet på jobben

gruble committed:
Tilpasset endringer gjort i API for bilder generert fra snøprofil-plott.

amish1188 commented on src/settings.ts:
```diff
@@ -73,8 +73,8 @@ export const settings: ISettings = {
-        TEST: 'https://test-api.regobs.no/v6',
-        //TEST: 'http://localhost:40001',
+        //TEST: 'https://test-api.regobs.no/v6',
+        TEST: 'http://localhost:5130',
```
dette er kanskje ikke ritkig?

amish1188 commented on src/app/components/observation/observation-image-carousel/observation-image-carousel.component.ts:
```diff
@@ -58,20 +58,6 @@ export class ObservationImageCarouselComponent {
 
   currentAttachmentData = computed(() => this.attachments()?.[this.attachmentIndex()]);
 
-  comment = computed(() => {
```
hmm skal vi ikke vise kommentarer?

amish1188 replied:
ok nvm. jeg ser du henter kommentar fra currentAttachmentData nå

amish1188 reviewed (changes requested):
Eneste som kanskje må fikses er TEST url som skal deployes. Tror ikke localhost er riktig url-en ;)

gruble committed:
Tilbakestilt test-api-url som ikke skulle vært endret

gruble committed:
Fikset en feil som gjorde at snøprofil-vedlegget ble returnert også når vi spesifiserte andre skjema enn snøprofil.

Fjernet alt av adaptive-elements fra modellen siden vi ikke får dette fra API v.6
